.gitignore

```diff
@@ -3,3 +3,22 @@ __pycache__/
 *.sw*
 .pytest_cache/
 config.ini
+
+# Distribution / packaging
+.Python
+build/
+develop-eggs/
+dist/
+downloads/
+eggs/
+.eggs/
+lib/
```

Comment:
Some of these are pointless. Please don't add entries defensively. Changes are expected to be reviewed before committing so any unintential additions should be noticed.

Reply:
Maybe just remove those entries? Some of those entries are for generated file from `python setup.py install`, `python setup.py sdist` or `python setup.py build`. But I guess those setup.py commands will not appear during development of lilac?

Reply:
`setup.py` generates only `dist`, `sdist` and `build`. I don't know where others come from.

Reply:
They are generated by IDEA and cloud be deleted if not used.

Reply:
`*.egg-info` is generated from `python setup.py egg_info` (a command used by `python setup.py build` or `python setup.py install`).
`.eggs` is used to store packages specified in `setup_requires` if they are not installed beforehand.

In the updated commit (6f2479f8d9ac2e088bda62112a783bf63b08ef1b), I keep entries that might be generated by setuptools and remove others.